R/client.R

```diff
@@ -409,7 +409,7 @@ client_parse_r_versions <- function(txt) {
 
 client_parse_headnodes <- function(txt) {
   dat <- strsplit(txt, "\n")[[1]]
-  stopifnot(all(grepl("^fi--", dat)))
+  stopifnot(all(grepl("^fi--|wpia-", dat)))
```

Comment:
```suggestion
  stopifnot(all(grepl("^(fi--|wpia-)", dat)))
```

is the wpia prefix a single prefix? oh dear

Reply:
"wpia-" is the new "fi--" - apparently it is funding specific.

Reply:
I understand the fi/wpia part, but surprised at the hyphen number is all